Paymetheus/ViewModels/StakeMiningViewModel.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Paymetheus.Decred.Wallet;
+using Paymetheus.Framework;
+using Grpc.Core;
+using Paymetheus.Decred.Util;
+using System;
+using System.Windows;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace Paymetheus.ViewModels
+{
+    public class StakeMiningViewModel : ViewModelBase
+    {
+        public StakeMiningViewModel() : base()
+        {
+            FetchStakeInfoCommand = new DelegateCommand(FetchStakeInfoAction);
+            FetchStakeInfoCommand.Execute(null);
+        }
+
+        private StakeInfoProperties _stakeInfoProperties;
+        public StakeInfoProperties StakeInfoProperties
+        {
+            get { return _stakeInfoProperties; }
+            internal set { _stakeInfoProperties = value; RaisePropertyChanged(); }
+        }
+
+        private StakeDifficultyProperties _stakeDifficultyProperties;
+        public StakeDifficultyProperties StakeDifficultyProperties
+        {
+            get { return _stakeDifficultyProperties; }
+            internal set { _stakeDifficultyProperties = value; RaisePropertyChanged(); }
+        }
+
+        public ICommand FetchStakeInfoCommand { get; }
+
+        private async void FetchStakeInfoAction()
+        {
+            try
+            {
+                StakeInfoProperties = await App.Current.Synchronizer.WalletRpcClient.StakeInfoAsync();
+                StakeDifficultyProperties = await App.Current.Synchronizer.WalletRpcClient.StakeDifficultyAsync();
```

Comment:
Calling these two separately is racy.  They need to be done under the same RPC and the server response must be calculated atomically as well.